pyreal/transformers/scale.py

```diff
@@ -0,0 +1,124 @@
+from sklearn.preprocessing import MinMaxScaler as SklearnMinMaxScaler
```

Comment:
Please run all tutorial scripts one more time so the saved output does not include errors